src/thunderbots/software/ai/primitive/directwheels_primitive.cpp

```diff
@@ -0,0 +1,77 @@
+#include "ai/primitive/directwheels_primitive.h"
+
+const std::string DirectWheelsPrimitive::PRIMITIVE_NAME = "DirectWheels Primitive";
+
+DirectWheelsPrimitive::DirectWheelsPrimitive(unsigned int robot_id, int wheel0_power,
+                                             int wheel1_power, int wheel2_power,
+                                             int wheel3_power, double dribbler_rpm)
+    : robot_id(robot_id),
+      wheel0_power(wheel0_power),
+      wheel1_power(wheel1_power),
+      wheel2_power(wheel2_power),
+      wheel3_power(wheel3_power),
+      dribbler_rpm(dribbler_rpm)
```

Comment:
I think we should throw an exception here if any of the wheel powers are out of bounds?

Reply:
See above